src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,28 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int matchOutcome;
```

Comment:
Don’t forget to check yourself with checklist before submitting task
 https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Reply:
what about third point ?